content/4_Gold/Divisibility.mdx

```diff
@@ -228,6 +229,22 @@ public class Divisors {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from math import sqrt
+
+n = int(input())
+for i in range(n):
+	div_num = 0
+	x = int(input())
+	for j in range(1, int(sqrt(x) + 1)):
+		if x % j == 0:
+			div_num = div_num + 1 if j ** 2 == x else div_num + 2
+	print(div_num)
```

Comment:
can you make the var names consistent w/ the other solutions?

Reply:
okay